docs/website-style.md

```diff
@@ -11,83 +11,72 @@ We're still building out custom CSS functionality with the MyST engine.
 Follow and comment on the issues linked below to help us improve it!
 :::
 
-## Use content blocks
+## Defining a Style Sheet
 
-:::{warning} CSS class support is very limited
-Currently, you can only use CSS classes that are pre-loaded by MyST from Tailwind CSS, or defined in the HTML theme (see below for examples of both).
-See these issues to track some of this:
+Default MyST themes like the book theme and article theme support bundling a custom [style-sheet](https://en.wikipedia.org/wiki/CSS). This can be used to introduce custom CSS styling to your website. To include a custom CSS file as part of your website build, you can define the @template-site-myst-book-theme-style option, e.g.
```

Comment:
Could we just link to the "default Myst themes" docs and remove the "like the book theme and..." language to keep it shorter?

Reply:
Yes, let's be more readable